packages/pages/src/util/runtime.ts

```diff
@@ -9,22 +11,26 @@ class Runtime {
   version: string;
 
   constructor() {
-    if (typeof process !== "undefined") {
-      this.name = "node";
-      this.version = process.versions.node;
-      this.isServerSide = true;
-    } else if (typeof window !== "undefined" && !("Deno" in window)) {
-      this.name = "browser";
-      this.version = navigator.userAgent;
-      this.isServerSide = false;
-    } else {
+    if (isDeno) {
       this.name = "deno";
       this.version = "";
-      if (typeof window !== "undefined") {
+      if (isBrowser) {
         this.version = (window as any).Deno?.version.deno || "";
       }
       this.isServerSide = true;
```

Comment:
I might be misunderstanding but it looks like `runtime.isServerSide` can be true when `isBrowser` is true since we always set isServerSide to true here?

Reply:
Javascript can be run server side in the deno environment and then included on the rendered web page to be viewed in the browser

Reply:
When it's included on the rendered web page, wouldn't the user want isServerSide to be false though?

Reply:
If you're fine as is I'm ok with it

Reply:
I would like to limit the scope of my changes to addressing the false positive possible in a node environment, and not change the existing functionality